### DISCUSSION TITLE
Two possible cosmetic bugs

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [outline/outline#7258] [General]
### Unpublishing a document temporarily disappears options
1. Publish a document.
2. Click the three dots in the upper-right corner for the document.
3. Click **Unpublish**.  The editing and publishing options immediately disappear from the top of the document.

Workaround: reloading the page makes the editing and publishing options reappear.

### Removing explicit permissions from a workspace administrator to a collection temporarily prohibits viewing collection
1. Go to the Collections settings page as a workspace administrator (/settings/collections).
2. Give yourself explicit access to the collection.
3. Remove that access.  The collection immediately disappears from the collection list on the page.

Workaround: reloading the page makes the collection reappear.

**Comment by tommoor** [comment]
> Unpublishing a document temporarily disappears options


Fixed in 8186e23d45eef886e42db99ff141bc10a769abcb, thanks for reporting

> Removing explicit permissions from a workspace administrator to a collection temporarily prohibits viewing collection

Noted, but not yet resolved
